### ISSUE TITLE
Upgrade to Teleport 2.0

### ISSUE THREAD

**Issue by kontsevoy** [gravitational/teleconsole#16]
[Teleport 2.0](https://github.com/gravitational/teleport/milestone/8) is coming out soon. When it does, vendor it into Teleconsole 0.4

**Comment by shadiakiki1986** [comment]
Looks like Teleport 2.0 was out since April with a [Teleport 2.3.5](https://github.com/gravitational/teleport/milestone/19) available

**Comment by kontsevoy** [comment]
@shadiakiki1986 correct. Unfortunately it looks like Teleport 2.x series is structured in a way that would break backwards compatibility, i.e. 2.x based clients won't be able to talk to 1.x based teleconsole server. I need to think about this a bit more.

**Comment by shadiakiki1986** [comment]
It's a breaking change for security, with an available remedy. The user
that is depending on the private key could either transfer the public key
to the server and use it, or could extract the public key from the private
key and use it. And if the user insists on sticking with the private key,
they could always stick with version 1 as well.
